mlflow/fastai/__init__.py

```diff
@@ -67,22 +65,30 @@ def get_default_conda_env(include_cloudpickle=False):
         :caption: Output
 
         conda environment: {'name': 'mlflow-env',
-                            'channels': ['defaults', 'conda-forge'],
-                            'dependencies': ['python=3.7.5', 'fastai=1.0.61',
+                            'channels': ['defaults', 'conda-forge', 'fastai',
+                                         'pytorch'],
+                            'dependencies': ['python=3.7.5', 'fastai=2.2.7',
+                                             'pytorch=1.8.0, 'torchvision=0.9.0',
                                              'pip', {'pip': ['mlflow']}]}
     """
 
     import fastai
+    import torch
+    import torchvision
 
     pip_deps = None
     if include_cloudpickle:
         import cloudpickle
 
         pip_deps = ["cloudpickle=={}".format(cloudpickle.__version__)]
     return _mlflow_conda_env(
-        additional_conda_deps=["fastai={}".format(fastai.__version__)],
+        additional_conda_deps=[
+            "fastai={}".format(fastai.__version__),
+            "pytorch={}".format(torch.__version__),
+            "torchvision={}".format(torchvision.__version__),
+        ],
```

Comment:
For context, MLflow is moving towards a conda-free environment (in favor of pip). Could you pull from the master branch, update these deps as pip_deps to resolve conflicts? Thanks!